playbooks/install_vision_dependencies.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: pyro-vision dependecies
+  hosts: raspberry
+
+  pre_tasks:
+    - name: install some dependecies
```

Comment:
Maybe I'd also suggest to put a better name for that task, it's a bit vague. Something like "install dependecies needed for pyro-vision module" or something also these lines (to specify what these libs are here for). What do you think ?